include/data_structures/message.hpp

```diff
@@ -10,22 +10,22 @@ using namespace std;
 /*******************************************/
 /**************** Message_t ****************/
 /*******************************************/
-struct Message_t{
+struct message_t{
 
-  Message_t(){}
-  Message_t(float i_value)
-   :value(i_value){}
+    message_t(){}
+    message_t(float i_value)
+    :value(i_value){}
 
-  	float   value;
+  	    float   value;
 
-  void clear() {
-   Message_t(); 
+    void clear() {
+        message_t(); 
   }
```

Comment:
curly brackets should end in the same column.

Reply:
Done as requested, Thanks